pkg/gui/options_menu_panel.go

```diff
@@ -79,7 +80,7 @@ func (gui *Gui) handleCreateOptionsMenu() error {
 	})
 
 	return gui.c.Menu(types.CreateMenuOptions{
-		Title:      strings.Title(gui.c.Tr.LcMenu),
+		Title:      cases.Title(language.Und).String(gui.c.Tr.LcMenu),
```

Comment:
rather than import the cases package, could we instead define a `gui.c.Tr.Menu` key which is manually titleized? Likewise with the other place we need to do this